### ISSUE TITLE
Ressource Booking - Book Function

### ISSUE THREAD

**Issue by qy21gafy** [qy21gafy/AMOSSharedDeskFeatureBoard#17]
## User story
1. As a user
2. I want / need to be able to book a ressource returned from the search function
3. So that I have a desk when I want to work in the office

## Acceptance criteria
*  Book function should be initialized by pressing the book button next to a ressource from the search result overview
* Pressing the book button should apply the booking in the database and provide the user with some kind of feedback regarding if the booking of that ressource was successful
* After booking the ressource it should appear under the "Booking Overview" for that user
* After booking the ressource it should not be able to be booked again for the same date and time

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
